### PR TITLE
Add models

### PR DESCRIPTION
**Pull request by bruce-willis**
* Add base audit models
* Update XF package in solutions
* Add ParseXmlHelper
* Add offline access to speaker from assembly
* Change caching strategy (now scroll works faster)

### CODE REVIEW

File: DotNetRu.Android/DotNetRu.Android.csproj [hunk 1]
```diff
@@ -109,8 +109,7 @@
       <Private>True</Private>
     </Reference>
     <Reference Include="FormsViewGroup, Version=2.0.0.0, Culture=neutral, processorArchitecture=MSIL">
-      <HintPath>..\packages\Xamarin.Forms.2.3.2.127\lib\MonoAndroid10\FormsViewGroup.dll</HintPath>
-      <Private>True</Private>
```
Comment (pfedotovsky) [inline DotNetRu.Android/DotNetRu.Android.csproj]: Why did you remove <Private>True</Private>?
With that attribute, the assembly won't be copied to the output directory.

File: DotNetRu.Clients.Portable/Helpers/ParseXmlHelper.cs [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+            writer.Write(@this);
+            writer.Flush();
+            stream.Position = 0;
+            return stream;
+        }
+
+
+        public static T ParseXml<T>(this string @this) where T : class
+        {
+            var reader = XmlReader.Create(@this.Trim().ToStream(), new XmlReaderSettings() { ConformanceLevel = ConformanceLevel.Document });
```
Comment (pfedotovsky) [inline DotNetRu.Clients.Portable/Helpers/ParseXmlHelper.cs]: You can just use XmlReader.Create(new StringReader(YourStringValue));
So ToStream() method is not needed

File: DotNetRu.Clients.Portable/ViewModel/SpeakersViewModel.cs [hunk 3]
```diff
@@ -70,7 +74,44 @@ orderby speaker.FullName
 		public ICommand LoadSpeakersCommand =>
 			loadSpeakersCommand ?? (loadSpeakersCommand = new Command(async (f) => await ExecuteLoadSpeakersAsync((bool) f)));
 
-		async Task<bool> ExecuteLoadSpeakersAsync(bool force = false)
+	    private static IEnumerable<Speaker> GetSpeakers()
```
Comment (pfedotovsky) [inline DotNetRu.Clients.Portable/ViewModel/SpeakersViewModel.cs]: Return List<Speaker> - it's more efficient as the caller won't need to enumerate the collection

File: DotNetRu.Android/DotNetRu.Android.csproj (hunk 1 above)
Comment (bruce-willis) [reply DotNetRu.Android/DotNetRu.Android.csproj]: I didn't do it manually. It's removed automatically when new version of XF was installed

File: DotNetRu.Clients.Portable/ViewModel/SpeakersViewModel.cs [hunk 4]
```diff
@@ -70,7 +74,44 @@ orderby speaker.FullName
...
+	    {
+	        var assembly = Assembly.Load(new AssemblyName("DotNetRu.DataStore.Audit"));
```
Comment (pfedotovsky) [inline DotNetRu.Clients.Portable/ViewModel/SpeakersViewModel.cs]: Seems a bit overkill to load an assembly just to get resource file. You can just put the file somewhere and load it.

File: DotNetRu.Clients.Portable/Helpers/ParseXmlHelper.cs (hunk 2 above)
Comment (bruce-willis) [reply DotNetRu.Clients.Portable/Helpers/ParseXmlHelper.cs]: I just copy this class from CPOD 🤔

Comment (pfedotovsky) [reply DotNetRu.Clients.Portable/Helpers/ParseXmlHelper.cs]: @bruce-willis then I've overlooked it there. But let's improve the code here :)

File: DotNetRu.Android/DotNetRu.Android.csproj (hunk 1 above)
Comment (pfedotovsky) [reply DotNetRu.Android/DotNetRu.Android.csproj]: @bruce-willis OK, there might be a good reason for it. Let's keep it as it is for now
